src/main/features/claim/views/defendant-organisation-details.njk

```diff
@@ -24,11 +24,13 @@
             ],
             addressSection = {
               heading: 'Organisation address',
-              hint: 'Enter the organisation’s main office or an address that has a connection with the claim.',
+              hint: 'Enter the organisation’s main office or an address that has a connection with the claim.
+                      The address must be in England or Wales.',
```

Comment:
The same here.

Reply:
@geek-soft: new line addressed. Made it single line.